packages/react-components/src/__tests__/mail.test.ts

```diff
@@ -6,8 +6,18 @@ import {
 } from '../mail';
 
 describe('createMailTo', () => {
-  it('generates a mailto link', () => {
-    expect(new URL(createMailTo('test@example.com')).protocol).toBe('mailto:');
+  it('generates a mailto link, when an single email is provided', () => {
+    const mailTo = new URL(createMailTo('test@example.com'));
+    expect(mailTo.protocol).toBe('mailto:');
+    expect(mailTo.href).toBe('mailto:test@example.com');
+  });
+
+  it('generates a mailto link, when multiple emails are provided', () => {
+    const mailTo = new URL(
+      createMailTo(['test@example.com', 'test@example.com']),
+    );
+    expect(mailTo.protocol).toBe('mailto:');
+    expect(mailTo.href).toBe('mailto:test@example.com,test@example.com,');
```

Comment:
Does the format require a trailing comma for multiple addresses?

Reply:
it does

Reply:
mailto:test@example.com,test@example.com,

Reply:
ohh, the final one... :D it's not, ofc

Reply:
Yeah both seem to work. I think having a trailing comma is a bug

Reply:
I thought about it, but was on a thight schedule :)

Reply:
This test should fail

Reply:
I'm pushing new code :)